definition-map.js

```diff
@@ -75,7 +75,56 @@ const QUERIES = {
             includeCorrectAnswers: "Flag to include questions the user previously got correct.",
             quizzes: "The IDs of the Quizzes user selects."
         }
-    }
+    },
+    Forums: {
+        metadescription: "This query returns a forum object. A forum is any area you can add a discussion to. For example, assignments, forum page type, discussions below the course on each section.",
+        args: {
+            courseId: "The ID of the course. This arguement is required", 
+            clientId: "The ID of the client"
+        }
+    },
+    Threads: {
+        metadescription: "This query returns a list of threads.",
+        args: {
+            courseId: "The ID of the course", 
+            clientId: "The ID of the client",
+            forumId: "The ID of the forum. This argument is required",
+            commentableType: "A commentable type can be thread, widgetThread, assignmentSubmission, assignment or a discussionBoard",
```

Comment:
To answer your question, don't need to list out the `commentableType`s. It's an enum, so the rendered documentation should render a link to a list of valid values. For the description, would say something like "The type of thread requested, based on the forum's location, e.g. whether it's a course discussion, discussion widget, etc" (would def. word it better than that as well)